app/code/Scandiweb_Test/Setup/Patch/Data/Test.php

```diff
@@ -31,43 +32,36 @@
 /**
  * Create Migration product class
  */
-class Test implements DataPatchInterface
+class Scandiweb_Test implements DataPatchInterface
 {
     /**
      * @var ProductInterfaceFactory
      */
     protected ProductInterfaceFactory $productInterfaceFactory;
-
```

Comment:
Why you removed this line, you need to add newline between prop declarations, please.